cypress/integration/Button.spec.ts

```diff
@@ -17,13 +17,13 @@ describe('Button', () => {
     it('should render the correct text', () => {
       cy.get('button')
         .first()
-        .should('contain', 'Primary');
+        .should('contain', 'Click Me');
     });
   });
 
   context('given delete buttons are rendered', () => {
     beforeEach(() => {
-      h.stories.load('Components/Buttons/Button/React', 'Delete');
+      h.stories.load('Components/Buttons/Button/React', 'DeleteStory');
```

Comment:
@jamesfan had a [good writeup](https://github.com/Workday/canvas-kit/pull/1127#issuecomment-869947485) of why this is necessary. After doing some digging in the Storybook [source code](https://github.com/storybookjs/storybook/blob/master/addons/docs/src/mdx/mdx-compiler-plugin.js#L23) it looks like they sanitize/rename stories if the name of the story 1. starts with a number, or 2. uses a reserved name in javascript (e.g. `delete`, `default`, `const`, etc.). I started to put some fixes into [`h.stories.load`](https://github.com/Workday/canvas-kit/blob/master/cypress/helpers/stories.ts#L10), but I quickly realized this is probably something that should go in [`cypress-storybook`](https://github.com/NicholasBoll/cypress-storybook).

@NicholasBoll I started to work on a PR in `cypress-storybook` for this, but ran into some issues due to how Storybook treats CSF vs. MDX stories. Unfortunately it seems like Storybook treats these two inconsistently:

For CSF stories:
- Stories with reserved keyword names (e.g. `Delete`) are not touched (URL is unaffected)
- Stories with number prefixed names have `Story` appended (resulting in `-story` to be appended in the URL)

For MDX stories:
- Stories with reserved keyword names (e.g. `Delete`) have `Story` appended (resulting in `story` being appended to the URL)
- Stories with number prefixed names have `_` prepended. 

I started to add logic similar to https://github.com/storybookjs/storybook/blob/master/addons/docs/src/mdx/mdx-compiler-plugin.js#L23 to [`loadStory`](https://github.com/NicholasBoll/cypress-storybook/blob/411184e81a6555ea1d6cac6572408aa45c8e5fa7/cypress.js#L7). However, in Storybook this logic is only run for MDX, so I realized that this would cause problems due to the inconsistency of treatment between CSF & MDX. AFAIK, `cypress-storybook` has no way to know whether the story is MDX or CSF based, so I got stuck. Pretty frustrating on Storybook's part, to be honest. 

I decided to just append `Story` in this instance for now and we can address this in `cypress-storybook` once we figure out the correct approach.